controller/cmd/tap/main.go

```diff
@@ -34,7 +34,8 @@ func main() {
 		restrictToNamespace = *controllerNamespace
 	}
 	k8sAPI := k8s.NewAPI(
-		clientSet,
+		k8sClient,
+		spClient,
```

Comment:
Right, and related to my comment about `NewSpClientSet`, let's just pass in nil for this argument in every `NewAPI` call, until we have a process that's actually going to watch and fetch SPs.